src/NodaTime.Web/Views/Shared/_Layout.cshtml

```diff
@@ -129,7 +129,6 @@
     <script src="/js/vendor/modernizr.js"></script>
     <script src="/js/vendor/jquery.js"></script>
     <script src="/js/foundation/foundation.js"></script>
-    <script src="/js/foundation/foundation.accordion.js"></script>
```

Comment:
Make the corresponding change in `src/NodaTime.Web/bundleconfig.json` so that we remove it from the minified JS (site.min.js) too?

Reply:
Sure, will do.